libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1728,3 +1746,52 @@ def generate_and_compare_both_version(input_installing_version, installed_versio
         else:
             LOGGER.info("Installing version is not higher than installed version.")
         return installing_version
+
+    @staticmethod
+    def create_namespace(master_node_list, namespace):
+        """
+        This method is used for creating custom namespace
+        param: master_node_list: node object for primary node
+        param: namespace: custom namespace [a-z][0-9] `-``characters
+        returns True, namespace created message
+        """
+        LOGGER.debug("The namespace to be created is %s", namespace)
+        master_node_list.execute_cmd(common_cmd.KUBECTL_CREATE_NAMESPACE.format(namespace),
+                                     read_lines=True)
+        resp = master_node_list.execute_cmd(common_cmd.KUBECTL_GET_NAMESPACE, read_lines=True)
+        if namespace in resp:
+            LOGGER.debug("The namespace is %s", resp)
+            return namespace
+        return False, f"Failed to create namespace: {resp}"
+
+    @staticmethod
+    def del_namespace(master_node_list, namespace):
+        """
+        This method is used for delete custom namespace
+        param: master_node_list: node object for primary node
+        param: namespace: custom namespace [a-z][0-9] `-``characters
+        returns True
+        """
+        LOGGER.debug("The namespace to be deleted is %s", namespace)
+        master_node_list.execute_cmd(common_cmd.KUBECTL_DEL_NAMESPACE.format(namespace),
+                                     read_lines=True)
+        resp = master_node_list.execute_cmd(common_cmd.KUBECTL_GET_NAMESPACE, read_lines=True)
+        LOGGER.debug("The namespace is %s", resp.pop(0))
+        if namespace in resp:
+            return False, f"Failed to del namespace {namespace}"
+        return True, f"Successfully deleted the NAMESPACE {namespace}"
+
+    @staticmethod
+    def aphanumeric_string_generator(size):
```

Comment:
We have it already. Check `random_string_generator` from `commons/utils/system_utils.py`

Reply:
those were either having lowercase or upper and lower case mix , we explicit need digits and lowercase only , uppercase is not supported.

Reply:
you can pass `chars=string.digits + string.ascii_lowercase` parameter to that function.